modules/arrow/test/geoarrow/convert-geoarrow-to-geojson.spec.ts

```diff
@@ -0,0 +1,246 @@
+const GEOARROW_ENCODINGS = [
+  'geoarrow.multipolygon',
+  'geoarrow.polygon',
+  'geoarrow.multilinestring',
+  'geoarrow.linestring',
+  'geoarrow.multipoint',
+  'geoarrow.point',
+  'geoarrow.wkb',
+  'geoarrow.wkt'
+];
+
+// a simple geojson contains one point
+const expectedPointGeojson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 2,
+        name: 'name2'
+      },
+      geometry: {
+        type: 'Point',
+        coordinates: [1, 1]
+      }
+    }
+  ]
+};
+
+// a simple geojson contains one linestring
+const expectedLineStringGeoJson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 1,
+        name: 'name1'
+      },
+      geometry: {
+        type: 'LineString',
+        coordinates: [
+          [0, 0],
+          [1, 1]
+        ]
+      }
+    }
+  ]
+};
+
+// a simple geojson contains one polygon
+const expectedPolygonGeojson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 1,
+        name: 'name1'
+      },
+      geometry: {
+        type: 'Polygon',
+        coordinates: [
+          [
+            [0, 0],
+            [1, 1],
+            [2, 2],
+            [0, 0]
+          ]
+        ]
+      }
+    }
+  ]
+};
+
+// a simple geojson contains one MultiPoint
+const expectedMultiPointGeoJson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 2,
+        name: 'name2'
+      },
+      geometry: {
+        type: 'MultiPoint',
+        coordinates: [
+          [1, 1],
+          [2, 2]
+        ]
+      }
+    }
+  ]
+};
+
+// a simple geojson contains one MultiLinestring
+const expectedMultiLineStringGeoJson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 2,
+        name: 'name2'
+      },
+      geometry: {
+        type: 'MultiLineString',
+        coordinates: [
+          [
+            [1, 1],
+            [2, 2]
+          ],
+          [
+            [3, 3],
+            [4, 4]
+          ]
+        ]
+      }
+    }
+  ]
+};
+
+// a simple geojson contains one MultiPolygon
+const expectedMultiPolygonGeojson = {
+  type: 'FeatureCollection',
+  features: [
+    {
+      type: 'Feature',
+      properties: {
+        id: 1,
+        name: 'name1'
+      },
+      geometry: {
+        type: 'MultiPolygon',
+        coordinates: [
+          [
+            [
+              [0, 0],
+              [0, 1],
+              [1, 1],
+              [1, 0],
+              [0, 0]
+            ]
+          ],
+          [
+            [
+              [2, 2],
+              [2, 3],
+              [3, 3],
+              [3, 2],
+              [2, 2]
+            ]
+          ]
+        ]
+      }
+    }
+  ]
+};
+
+test.skip('ArrowUtils#parseGeometryFromArrow', async (t) => {
```

Comment:
@lixun910 This test case crashes, haven't had time to debug yet.

Reply:
Update the `serializeArrowType()` and `deserializeArrowType()` functions to fix the issues.